hal/halx86/acpi/halpnpdd.c

```diff
@@ -153,6 +153,30 @@ HalpQueryInterface(IN PDEVICE_OBJECT DeviceObject,
                    IN PINTERFACE Interface,
                    OUT PULONG Length)
 {
+    // See /sdk/include/ddk/wdmguid.h.
+    const GUID GUID_ACPI_REGS_INTERFACE_STANDARD =
+        { 0x06141966, 0x7245, 0x6369, { 0x46, 0x2e, 0x4e, 0x65, 0x6c, 0x73, 0x6f, 0x6e } };
+    const GUID GUID_ACPI_PORT_RANGES_INTERFACE_STANDARD =
+        { 0xf14f609b, 0xcbbd, 0x4957, { 0xa6, 0x74, 0xbc, 0x00, 0x21, 0x3f, 0x1c, 0x97 } };
+
+    if (IsEqualIID(InterfaceType, &GUID_ACPI_REGS_INTERFACE_STANDARD))
+    {
+        DPRINT1("InterfaceType: %s\n", "GUID_ACPI_REGS_INTERFACE_STANDARD");
```

Comment:
No need for the %s thing

Reply:
That was an attempt at mergeable strings.
Removed.